about.html

```diff
@@ -0,0 +1,112 @@
+                        <li id="home"><a href="home.html">Program</a></li> 
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                    </ul>
+                </div>
+                <a href="#" id="campaign">WD Campaign</a>
+            </div>
+            
+        </nav>
+    </header>
+
+    <section class="s1">
+        <div class="home-about">
+            <img src="images/Icon - Menu.svg" id="open-menu" alt="">
+             <h2>"Hello! Change world"</h2>
+             <p>Web development course using HTML, CSS, and JavaScript</p>
+            <div class="p-about" >
+                <p>This course offers all the details that you require to fully learn about the world of web programming and coding.
+                    You will learn all about web programming from scratch and will be able to design your own websites by the end of this course.
+                </p>
+            </div>
+            <h3>Please contact us via email for any further questions about the web development course. </h3>
+            <div class="a-about"><a href="mailto:odai.alazzeh@gmail.com">odai.alazzeh@gmail.com</a></div>
+        </div>
+    </section>
+
+    <section class="about-logo">
+        <div>
+            <h2>Web development course using HTML,CSS, and JavaScript 2022 logo</h2>
+            <p>The logo of WD Course using HTML,CSS, and JavaScript 2022 was decided through the logo 
+                competition from 24. October to 25. October.
+            </p> 
+        </div>
+       <div class="left">
+            <h2><span>WD</span> Course </h2>
+            <h3>using HTML, CSS, and JavaScript</h3>
+        </div>
+    </section>
+
+    <section class="last-course">
+       <h2>See the past WD Course </h2>
+       <p>Take a look at the last two WD Course.</p>
+       <div class="img-course" id="img-course">
+        <div class="course1">
+            <h2>2020</h2>
+            <p>WD Course 2020</p>
+        </div>
+        <div class="course2">
+            <h2>2021</h2>
+            <p>WD Course 2021</p>
+        </div>
+       </div>
+    </section>
+
+    <footer>
+        <div class="footer-up">
+            <h2>Partner</h2>
+            <ul>
+                <li><img src="images/mozilla.svg" alt=""></li>
+                <li><img src="images/Google.svg" alt=""></li>
+                <li><img src="images/Naver.svg" alt=""></li>
+                <li><img src="images/DaumKakao.svg" alt=""></li>
+                <li><img src="images/Airbnb.svg" alt=""></li>
+            </ul>
+        </div>
+        <div class="footer-down">
+            <div class="left">
+            <h2><span>WD</span> Course </h2>
+            <h3>using HTML, CSS, and JavaScript</h3>
+            </div>
+            <div class="right">
+            <h3>2022 Web development course. </h3>
+            <h3>Some Rights Reserved</h3>
+            </div>
```

Comment:
- [ ] For the footer part of the about page, kindly make it dark for desktop version and light in the mobile version, to match the design guidelines 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/198081979-e58138b7-3ffa-42ad-8f06-083c2c04959b.png)

## Design guidelines:
![image](https://user-images.githubusercontent.com/50721479/198082016-b8642528-05fe-4887-af37-ebe1191e2ce7.png)